editor/plugins/gizmos/navigation_link_3d_gizmo_plugin.cpp

```diff
@@ -70,59 +70,101 @@ void NavigationLink3DGizmoPlugin::redraw(EditorNode3DGizmo *p_gizmo) {
 
 	p_gizmo->clear();
 
-	// Draw line between the points.
-	Vector<Vector3> lines;
-	lines.append(start_position);
-	lines.append(end_position);
-
-	// Draw start position search radius
-	for (int i = 0; i < 30; i++) {
-		// Create a circle
-		const float ra = Math::deg_to_rad((float)(i * 12));
-		const float rb = Math::deg_to_rad((float)((i + 1) * 12));
-		const Point2 a = Vector2(Math::sin(ra), Math::cos(ra)) * search_radius;
-		const Point2 b = Vector2(Math::sin(rb), Math::cos(rb)) * search_radius;
-
-		// Draw axis-aligned circle
-		switch (up_axis) {
-			case Vector3::AXIS_X:
-				lines.append(start_position + Vector3(0, a.x, a.y));
-				lines.append(start_position + Vector3(0, b.x, b.y));
-				break;
-			case Vector3::AXIS_Y:
-				lines.append(start_position + Vector3(a.x, 0, a.y));
-				lines.append(start_position + Vector3(b.x, 0, b.y));
-				break;
-			case Vector3::AXIS_Z:
-				lines.append(start_position + Vector3(a.x, a.y, 0));
-				lines.append(start_position + Vector3(b.x, b.y, 0));
-				break;
-		}
-	}
+	uint32_t points_in_octant = 4;
```

Comment:
This is a magic number corresponding to the quality / smoothness of the circle, right? If so, it should be static constexpr, and ideally be annotated that it can be changed to adjust the circle smoothness.

Reply:
True, a comment wouldn't hurt. Thanks